docs/import.md

```diff
@@ -25,15 +30,45 @@ This method uses **remixd** - the remix daemon.  Please go to the [remixd tutori
 Importing from GitHub
 ---------------------
 
-It is possible to import files directly from GitHub with URLs like
-`https://github.com/<owner>/<repo>/<path to the file>`.
+It is possible to import files directly from GitHub.
+
+```
+pragma solidity >=0.4.22 <0.6.0;
+
+import "https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/math/SafeMath.sol";
 
-![](images/a-old-tuto_importgit.png)
+```
 
 Importing from Swarm
 --------------------
 
-Files can be imported using all URLs supported by swarm. If you do not
-have a swarm node, then use swarm-gateways.net.
+Files can be imported using all URLs supported by swarm. 
+If you do not have a swarm node, then use swarm-gateways.net.
+
+```
+import 'bzz-raw://5766400e5d6d822f2029b827331b354c41e0b61f73440851dd0d06f603dd91e5';
+```
+
+Importing from IPFS
+--------------------
+
+Files can be imported from IPFS. 
+
+```
+import 'dweb:/ipfs/Qmdyq9ZmWcaryd1mgGZ4PttRNctLGUSAMpPqufsk6uRMKh';
```

Comment:
i think `ipfs://Qmdyq9ZmWcaryd1mgGZ4PttRNctLGUSAMpPqufsk6uRMKh` will work but not sure about
`dweb:/ipfs/Qmdyq9ZmWcaryd1mgGZ4PttRNctLGUSAMpPqufsk6uRMKh`

Reply:
right. 
ipfs://<ipfs_hash> is the right pattern for now. although we should add <ipfs_hash> as well. Same in swarm